### DISCUSSION TITLE
Thread count vs Core count

### DISCUSSION THREAD

**Discussion by vvavepacket** [madMAx43v3r/chia-plotter#278] [General]
What should we use in the number of threads?

I have a 5800X and it has 8 real cores and 16threads.

What should we use and supply to the parameter? Which is faster?

**Comment by madMAx43v3r** [comment]
try 8 or 12

**Reply by w4d3j** [reply]
Why not 16?  

Now I'm going to have to run a few with fewer threads to see if it somehow speeds things up lol

**Comment by maxchock** [comment]
Hi, do the formula as follow applied?
8 core + 16 thread / 2  = 12 thread.

**Comment by vvavepacket** [comment]
@maxchock  Whats the reason for getting the extra threads (12 thread)? Any mathematical or
hardware reason or link to back that up or just hearsay?

**Reply by maxchock** [reply]
hi, it was based on a blogger post, here is the link

https://thechiafarmer.com/2021/05/05/how-many-plots-can-i-make-a-day/
